syft/frameworks/torch/tensors/abstract.py

```diff
@@ -17,7 +17,30 @@ def wrap(self):
 
     def serialize(self,
                   compress=True,
-                  compress_scheme=sy.serde.LZ4):
+                  compress_scheme=0):
+        """This convenience method serializes the tensor on which it's called.
+
+            This is the high level convenience function for serializing torch tensors.
+            It includes three steps, Simplify, Serialize, and Compress as described
+            in serde.py
+
+            Args:
+                self (AbstractTensor): the tensor to be serialized
+
```

Comment:
Empty lines between args are not needed.

Reply:
fixed